src/algorithms/math/is_prime.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Checks if a num is prime or not
+ * @param  {Number} num number to check
+ * @return {Boolean} true if num is prime, false if num is not prime
+ *
+ * References: https://javascript.plainenglish.io/11-mathematical-algorithms-in-modern-javascript-bce71318e2da
+ */
+const isprime = (num) => {
+  const limit = Math.floor(Math.sqrt(num));
```

Comment:
before this you have to verify if the number are greater or equal to 1 or if is a integer

  if (num <= 1 || !Number.isInteger(num)) {
    return false;
  }